android/app/src/main/java/com/chorestory/adapter/ChildHomeAdapter.java

```diff
@@ -0,0 +1,50 @@
+package com.chorestory.adapter;
+
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChildHomeAdapter extends FragmentStatePagerAdapter {
+
+    private List<Fragment> fragmentList;
+    private List<String> fragmentTitleList;
+
+    public ChildHomeAdapter(FragmentManager fragmentManager) {
```

Comment:
Why are we keeping this around? Is there some library that is going to call this? Let's remove it if not to discourage creating empty lists.
I don't see this being called manually anywhere.